.github/workflows/functional.yaml

```diff
@@ -15,26 +15,14 @@ jobs:
     steps:
     - uses: actions/checkout@v3
 
-    - name: Create k8s Kind Cluster
-      uses: helm/kind-action@v1.5.0
-
-    - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v2
+    - name: Set up Helm
+      uses: azure/setup-helm@v3
       with:
-        python-version: ${{ matrix.python-version }}
+        version: v3.11.3
```

Comment:
I guess we probably want to remove the python-version stuff in the matrix now, but that is a tidy up really.